server/src/main/java/com/thoughtworks/go/server/controller/JobController.java

```diff
@@ -99,12 +100,12 @@ public JobController() {
         this.jobAgentMetadataDao = jobAgentMetadataDao;
     }
 
-    @RequestMapping(value = "/tab/build/recent", method = RequestMethod.GET)
-    public ModelAndView jobDetail(@RequestParam("pipelineName") String pipelineName,
-                                  @RequestParam("label") String counterOrLabel,
-                                  @RequestParam("stageName") String stageName,
-                                  @RequestParam("stageCounter") String stageCounter,
-                                  @RequestParam("jobName") String jobName) throws Exception {
+    @RequestMapping(value = "/tab/build/detail/{pipelineName}/{label}/{stageName}/{stageCounter}/{jobName}", method = RequestMethod.GET)
```

Comment:
changed **/tab/build/recent** to match external api endpoint **/tab/build/detail/**